satpy/Combined_colormaps.ipynb

```diff
@@ -0,0 +1,541 @@
+{
```

Comment:
<p><code>sencor</code>  -&gt; <code>sensor</code></p><p> </p><p>And should this section point to something in the main documentation as a "For more information see..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>